src/Rooms.js

```diff
@@ -57,7 +68,7 @@ class Rooms extends React.Component {
                     {item.roomType}
                   </span>
                   <span className="date">
-                    <Timestamp time={(item.createTms)/1000} format="date" />
+                    {<Timestamp time={(item.createTms)/1000} format="date" />}

```

Comment:
Because the given value of createTms (e.g. 1491239577417) is in miliseconds and react-timestamp needs to receive seconds in order to convert them to real date.